packages/react/src/side-nav/SideNavLinkList.tsx

```diff
@@ -7,11 +7,13 @@ import { hasNestedActiveItem } from './utils';
 export type SideNavLinkListProps = {
 	activePath: string | undefined;
 	items: SideNavProps['items'];
+	nestedItemsVariant: SideNavProps['nestedItemsVariant'];
```

Comment:
I wonder if something like `nestedItemsShow` or `subItemsShow` with values like `always` and `whenVisited` might be clearer?

Reply:
For a bit of context and insight into my thought process on this, it started out as:

```ts
nestedItemsVariant?: 'always-open' | 'open-on-nav'  | 'toggle-open-close';
```

The  `'toggle-open-close'` was the variant with the link + accordion style expand/collapse button. Although this variant has been de-scoped, I checked with the designers and it will likely return in some form later on. Due to the functional change with the accordion style (over the link only style), I chose to hang onto the word `variant`. Whatever we choose, I think we should setup for this extension.

Reply:
Good context. `onToggle` also supports your other scenario. I think pairing the prop name with the value as a "this happens when" structure makes it much easier to understand imo

Reply:
>pairing the prop name with the value as a "this happens when" structure

Interesting. I've always taken the declarative approach. `activePath` is this string, `items` is this array, `nestedItemsVariant` is this variant. Ideally `variant` would be enough here but I feel `nestedItems` acts as a TLDR so you can skip the rest if you're not using nested items. Having said all that, I don't mind changing this. I feel docs are the most important part of discoverability here, so I'm happy to roll with your suggestion. What do you think is the most appropriate?

```ts
subItemsShow?: 'always' | 'whenVisited' | 'onToggle';
```

Reply:
- [ ] `showSubLevel?: 'always' | 'whenActive'`

Reply:
- [ ] Migrate all "nested" text to "sub-level"